quantum-version/lib/lib.js

```diff
@@ -132,6 +132,12 @@ function processTags (content, version, versions, file) {
     }
   })
 
+  tags.filter(t => t.type() === 'bugfix').forEach(bugfixTag => {
+    if (bugfixTag.ps() !== version) {
+      bugfixTag.remove()
+    }
+  })
```

Comment:
Bugfixes were ignored by the versioning here but supported in the `api` sections meaning they were displayed in every version. We are now treating them the same way as 'updated'